bn-apps/memberships-management/membership-service/src/main/kotlin/net/corda/businessnetworks/membership/ConfigUtils.kt

```diff
@@ -0,0 +1,19 @@
+package net.corda.businessnetworks.membership
+
+import com.typesafe.config.Config
+import com.typesafe.config.ConfigFactory
+import net.corda.core.internal.div
+import java.io.File
+import java.nio.file.Paths
+
+/**
+ * Attempts to load application config from cordapps/config/membership-service.conf with fallback to membership-service.conf on the classpath
+ */
+object ConfigUtils {
+    fun loadConfig() : Config {
+        val propertiesFileName = "membership-service.conf"
+        val defaultLocation = (Paths.get("cordapps") / "config" / propertiesFileName).toFile()
+        return if (defaultLocation.exists()) ConfigFactory.parseFile(defaultLocation)
+            else ConfigFactory.parseFile(File(ConfigUtils::class.java.classLoader.getResource(propertiesFileName).toURI()))
```

Comment:
Yeps, which should work if there is a single membership service with a single membership-service.conf that is serving multiple business networks no?

Reply:
I see, so you envisage someone baking the membership-service.conf into the membership-service.jar. I suppose that is an option. I was worried people would start putting the membership-service.conf into their cordapps (like we did with leia.jar or cinsure.jar) and then if you had more of such cordapps running on your node (each one wanting to use different BNO) you could not tell which membership-service.conf would get picked.

Reply:
That makes sense. There is a single property that we are currently reading from the config file on member's side - bnoName. This property should go away once we support multiple BNOs. So we might end up without a need for a configuration file on member's side at all

Reply:
I think you will still need to link your cordapp to a business network somehow. So that you know what member list to look at when someone is trying to initiate a flow. Anyway, I suppose this pull request is not about multiple BNs anyway so I'm resolving this convo and we can continue on another branch :-)